olclient/entity_helpers/work.py

```diff
@@ -139,16 +139,13 @@ def rm_subjects(self, subjects, comment=''):
             data['subjects'] = list(set(data['subjects']) - set(subjects))
             return self.OL.session.put(url, json.dumps(data))
 
-        def delete(self, comment: str, confirm: bool = True) -> Optional[Response]:
-            edition_olids: List[str] = [edition_data.olid for edition_data in self.editions]
-            edition_count = len(edition_olids)
+        def delete(self, confirm: bool = True) -> Optional[Response]:
```

Comment:
Bring the comment back! That's important info. We set it as a get URL param on the delete endpoint.